simtools/util/tests.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+CONFIG_FILE_MSG = "Configuration file is not available"
+
+DB_CONNECTION_MSG = "There is not connection with DB"
+
+SIMTEL_MSG = "sim_telarray is not installed"
+
+
+def _collect_conftest_flag(name):
```

Comment:
I suggest to write explicitly "config_test" instead of "conftest" everywhere. Also in the name of "conftest.py".

Reply:
Not possible, the name conftest.py is hardcoded into pytest. Since this has to be the name of the file, I would keep in other places for consistency.

Reply:
really? how weird. Makes sense to keep consistent then and keep it everywhere.